tests/test_encoding.py

```diff
@@ -0,0 +1,17 @@
+"""Testing the parse_file function."""
```

Comment:
can we re-name this file to `test_entrypoint` and the module docstring accordingly - there's likely to be added more tests to it eventually.